KNet/KNet.API/Models/Entity.cs

```diff
@@ -2,8 +2,16 @@
 
 namespace KNet.API.Models
 {
-    public class Entity
+    public abstract class Entity
     {
+        public Entity()
+        {
+            Created = Modified = DateTimeOffset.UtcNow.ToUnixTimeSeconds();
+        }
         public Guid Id { get; set; }
+        public bool IsActive { get; set; }
+        public int Version { get; set; } = 1;
```

Comment:
I do not really understand this property, is this for identifying the entity version for this object?

Reply:
By setting IsActive to false you can soft-delete entities from the database.
Version is set to 1 by default, and is incremented whenever an entity is modified.